src/aoc2020/day2_copy.clj

```diff
@@ -0,0 +1,210 @@
+(ns aoc2020.day2_copy
+  (:require [aoc2020.util :refer [file->seq]])
+  (:require [clojure.string :as str]))
+
+(defn old-policy-pwd-categorizer
+  [policy-pwd-str]
+  (let [remove-char  (str/replace policy-pwd-str
+                       #"[^a-zA-Z0-9]" " ")
+        [lower-str upper-str letter pwd] (str/split remove-char #"\s+")
+        lower        (parse-long lower-str)
+        upper        (parse-long upper-str)
+        letter->char (seq letter)
+        pwd->char    (seq pwd)]
+    {:pwd-policy {:lower-limit  lower
+                  :upper-limit  upper
+                  :limit-letter letter->char}
+     :pwd        pwd->char}))
+
+
+(defn old-pwd-valid-check
+  [pwd-policy+pwd-map]
+  (let [{pwd-policy :pwd-policy
+         pwd        :pwd} pwd-policy+pwd-map
+        {lower-limit    :lower-limit
+         upper-limit    :upper-limit
+         [limit-letter] :limit-letter} pwd-policy
+        pwd-letter-cnt (frequencies pwd)
+        {letter-freq limit-letter :or {letter-freq 0}} pwd-letter-cnt
+        abv-lo-bnd?    (>= letter-freq
+                         lower-limit)
+        und-hi-bnd?    (<= letter-freq
+                         upper-limit)]
+    (and abv-lo-bnd? und-hi-bnd?)))
+
+(defn old-valid-password-cnt
+  [pwd-db-s]
+  (->> pwd-db-s
+    (mapv old-policy-pwd-categorizer)
+    (mapv old-pwd-valid-check)
+    (remove false?)
+    (count)))
+
+
+
+(defn Tob-policy-pwd-categorizer
+  [policy-pwd-str]
+  (let [remove-char  (str/replace policy-pwd-str
+                       #"[^a-zA-Z0-9]" " ")
+        [fir-pos-str sec-pos-str letter pwd] (str/split remove-char #"\s+")
+        fir-pos      (parse-long fir-pos-str)
+        sec-pos      (parse-long sec-pos-str)
+        letter->char (seq letter)]
+    {:pwd-policy {:first-pos    fir-pos
+                  :second-pos   sec-pos
+                  :limit-letter letter->char}
+     :pwd        pwd}))
+
+(defn Tob-pwd-policy->xor
```

Comment:
* Tob??
* Shouldn't named as a transform (->). This is inconsistent with clojure name (e.g., `or`, etc..)